package.json

```diff
@@ -12,6 +12,8 @@
         "eslint:fix": "eslint --ext .tsx --fix src/components"
     },
     "dependencies": {
+        "@cloudinary/react": "^1.13.1",
```

Comment:
If Cloudinary isn't being used, remove it as a dependency. Use `npm uninstall cloudinary`